docs/run-move-locally.md

```diff
@@ -0,0 +1,278 @@
+
+To run a local network with one validator node and create a local blockchain, change to the `libra` directory and run `libra_swarm`, as shown below:
+
+```
+$ cd libra
+$ cargo run -p libra_swarm -- -s
+```
+
+This command will take some time to run and it will perform the following actions:
+
+* Spawn a local network with one validator node on your computer.
+* Generate genesis transaction, mint key, and bootstrap configuration of the node.
+* Start an instance of the Libra CLI client; the client is connected to the local network.
+
+Upon successful execution of the command, you'll see the CLI client menu and the `libra%` prompt as shown below:
+
+```
+usage: <command> <args>
+
+Use the following commands:
+
+account | a
+Account operations
+query | q
+Query operations
+transfer | transferb | t | tb
+<sender_account_address>|<sender_account_ref_id> <receiver_account_address>|<receiver_account_ref_id> <number_of_coins> [gas_unit_price_in_micro_libras (default=0)] [max_gas_amount_in_micro_libras (default 100000)] Suffix 'b' is for blocking.
+Transfer coins (in libra) from account to another.
+dev
+Local Move development
+help | h
+Prints this help
+quit | q!
+Exit this client
+
+
+Please, input commands:
+
+libra%
+```
+
+For detailed instructions on working with a local cluster of validator nodes, refer to [Run a Local Network](run-local-network.md).
+
+### Create an Account
+
+Each Move module and resource type is hosted by a specific account address. For example, `LibraCoin` module is hosted by the account at address `0x0`. To import the `LibraCoin` module in other modules or transaction scripts, you would use `import 0x0.LibraCoin`.
+
+To host `MyModule`, create an account:
+
+```
+libra% account create
+>> Creating/retrieving next account from wallet
+Created/retrieved account #0 address 810abcc08dbed34ea15d7eb261b8001da6a62d72acdbf87714dd243a175f9b62
+
+```
+
+In the above output, 0 is the index of the account you just created, and the hex string is the address of that account. The index is just a convenient way to refer to this account locally, and it's also called `ref_id`. For more information on account creation, refer to [My First Transaction](my-first-transaction.md).
+
+The `create` command generates a local keypair. To create the account on the local blockchain, you'll need to mint money into the account, as shown below:
+
+```
+libra% account mintb 0 76
+>> Minting coins
+waiting ....transaction is stored!
+Finished minting!
+```
+To check whether the account was successfully created on the local blockchain, query the account balance. Note that the account balance you see in the output is in libra.
+
+```
+libra% query balance 0
+Balance is: 76.000000
+```
+
+### Compile Move Module
+
+To compile `my_module.mvir`, use the [dev compile](reference/libra-cli#dev-d-mdash-operations-related-to-move-transaction-scripts-and-modules) command.
+
+```
+libra% dev compile 0 <path to my_module.mvir> module
```

Comment:
Should we say what the "0" means?

Reply:
Added the description for 0 and updated the PR.